main.css

```diff
@@ -124,3 +119,150 @@ ul {
   align-items: center;
   margin-top: -15px;
 }
+
+.myrecentwork {
+  font-family: "roboto", sans-serif;
+  font-weight: 500;
+  font-size: 60px;
+  line-height: 52px;
+  text-align: left;
+  color: rgb(148, 144, 255);
+  display: flex;
+  justify-content: left;
+  align-items: center;
+  padding-right: -60px;
+  flex-wrap: wrap;
+  padding-left: 80px;
+  padding-top: 35px;
+}
+
+.picture {
+  display: flex;
+  flex-wrap: wrap;
+  padding: 50px;
+  justify-content: center;
+  align-items: center;
+  padding-left: 35px;
+  width: 335px;
+}
+
+.text {
+  color: rgb(255, 255, 255);
+  text-align: left;
+  padding-left: 35px;
+  display: flex;
+  align-items: left;
+  font-family: "roboto", sans-serif;
+  font-size: 50px;
+}
+
+.paragraph-text {
+  text-align: left;
+  padding-left: 35px;
+  padding-right: 35px;
+}
+
+.styling3 {
+  display: flex;
+  justify-content: space-around;
+  padding: 30px;
+}
+
+.button1 {
```

Comment:
- [ ] This button does not have the pseudo classes such as hover as required by the template. Please add the pseudo classes as seen in the image below 🏄 
![image](https://user-images.githubusercontent.com/45460192/170706407-49b9d9d0-1f78-47a4-9f2d-d980fd89bb56.png)